src/pages/BusinessLanding/Create/CouponForm.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useRef } from "react";
+import { Grid, TextField, Button, Typography } from "@mui/material";
+import axios from "axios";
+
+const defaultValues = {
+  item: "",
+  percentage: 0,
+  owner: "John's Candy"
+};
+
+const CouponForm = () => {
+    const [formValues, setFormValues] = useState(defaultValues);
+    const responseRef = React.useRef();
+    const handleInputChange = (e) => {
+    const { name, value } = e.target;
+    setFormValues({
+      ...formValues,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = (event) => {
+    const element = responseRef.current;
+
+    axios.post('http://localhost:8080/api/coupons', formValues)
```

Comment:
This will work for demo, but if it's better to save the hostname as a global constant and build the uri.